README.md

```diff
@@ -26,6 +31,24 @@ Scratch Addons by itself is just an addon loader. Its main tasks are:
 - Avoid addons from interfering with each other.
 - Avoid duplicate work from different addons.
 
-## How can I contribute?
+## Installation
+
+No build is required. After cloning the repository to your computer, just load the extension by following these steps.
+
+- Chrome
+  1. Open `chrome://extensions` to open the Extension Management page by typing it into your address bar.
+  2. Enable Developer Mode by clicking the toggle switch next to Developer mode.
+  3. Click the `LOAD UNPACKED` button on the top and select the extension directory. 
```

Comment:
I'd suggest you clarify it's the directory where `manifest.json` resides. Git usually puts downloaded folders inside their own folder (named the same) so it can get confusing.

Reply:
There you go.